mgmtsystem_hazard/__manifest__.py

```diff
@@ -13,7 +13,6 @@
         "security/ir.model.access.csv",
         "security/mgmtsystem_hazard_security.xml",
         "views/mgmtsystem_hazard.xml",
-        "views/mgmtsystem_hazard_hazard.xml",
```

Comment:
Why is this file being removed?

Reply:
The precomit script warned me that there was duplication of Id's. Then I checked that views/mgmtsystem_hazard_hazard.xml and views/mgmtsystem_hazard_origin.xml were identical files